osutils/client.go

```diff
@@ -0,0 +1,29 @@
+package osutils
+
+import (
+	"io/ioutil"
+	"os"
+)
+
+type OsClient interface {
+	Getenv(key string) string
+	ReadFile(path string) ([]byte, error)
+}
+
+type osClient struct {
```

Comment:
Not to add levels of abstraction, but how do you feel about adding an afero.fs here, or creating a seperate interface with afero. Creating utilities with it by default feels right to me.

Reply:
discussed in person -- can implement with afero in the future, but this is used by bots who don't need the abstraction, and the interface is just for mock testing